### PR TITLE
Prepare 0.0.342 release

### PR DESCRIPTION
**Pull request by chenz4027**
Update changes.md file

### CODE REVIEW

**Review by andreadecorte — APPROVED** [review]
lgtm

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.342 Dec 12 2023
+- Add `Search` and `Order` methods to List `/api/addons_mgmt/v1/clusters/{id}/node_pools`
```
Comment (andreadecorte) [inline CHANGES.md:6]:
```suggestion
- Add `Search` and `Order` methods to List `/api/clusters_mgmt/v1/clusters/{id}/node_pools`
```

Comment (chenz4027) [reply CHANGES.md]: Updated the clusters_mgmt

**Pull request merged**